zetaclient/orchestrator/mapping.go

```diff
@@ -0,0 +1,49 @@
+package orchestrator
```

Comment:
We can maybe move to `pkg`?

Reply:
I'd keep it here for now, not mature enough to be moved IMO